src/pages/search/searchPage.jsx

```diff
@@ -15,18 +15,19 @@ export default class SearchPage extends Component {
   componentDidMount() {
     //query string
 
-    const url = this.props.location.search;
-    console.log(url);
+    const url = this.props.location.pathname;
+
     const userRequest = url.split("=");
```

Comment:
Each route component from react-router gets a match prop that contains params, you don't have to destructure the url yourself

Reply:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/match.md